Wrappers/Python/cil/processors/TransmissionAbsorptionConverter.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think the header we decided on is different and you can find it [here](https://github.com/vais-ral/CCPi-Framework/issues/540#issuecomment-718794402)